plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -61,6 +64,45 @@ public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models,
                         collection.AddRange(result.Models);
                     }
                 }
+
+                var duplicatePathes = filePathsDict.Where(x => x.Value.Item1).Select(x => x.Key).ToList();
+                if (duplicatePathes.Any())
+                {
+                    foreach (var path in duplicatePathes)
+                    {
+                        var newFileModels = collection.Where(fm => string.Equals(fm.File, path, FilePathComparer.OSPlatformSensitiveStringComparison)).Select(
```

Comment:
>collection.Where [](start = 44, length = 16)

What about fix result.TreeItems and result.Models before add to treeMaping and collection? thus to avoid add it then remove it for duplicated items.